### PR TITLE
Beautify note photo cards

### PR DESCRIPTION
**Pull request by MattRice12**
Moderate styling edits to the notes and photos show pages. Waiting to see what other style edits were made to keep consistency.

### CODE REVIEW

**Comment by kate-walters** [comment]
Everything looks like it's written correctly 👍

**Comment by Langster2323** [comment]
Looks good! 👍

File: features/category_traversal.feature [hunk 1]
```diff
@@ -6,7 +6,6 @@ Background: An Admin Logs In
     And I fill in "Email" with "admin@example.com"
     And I fill in "Password" with "password"
     And I press "Submit"
-    Then I should see "Create Note"
```
Comment (rluck0419) [inline features/category_traversal.feature]: was this necessary?

Comment (MattRice12) [inline features/category_traversal.feature]: Not sure why that's still deleted. I thought I put it back.

File: features/category_creation.feature [hunk 2]
```diff
@@ -6,7 +6,6 @@ Background: An Admin Logs In and Goes to Category Index
     And I fill in "Email" with "admin@example.com"
     And I fill in "Password" with "password"
     And I press "Submit"
-    Then I should see "Create Note"
```
Comment (rluck0419) [inline features/category_creation.feature]: this too?

File: features/user_authentication.feature [hunk 3]
```diff
@@ -7,6 +7,5 @@ Feature: As a User, In order to use the app, I would like to be able to login
     And I fill in "Email" with "user@example.com"
     And I fill in "Password" with "password"
     And I press "Submit"
-    Then I should see "Create Note"
```
Comment (rluck0419) [inline features/user_authentication.feature]: and this one?

**Pull request merged**